Master is failing

https://travis-ci.org/opencontainers/image-spec/builds/209076959
```
The command "make test" exited with 0.
29.17s$ make docs
Unable to find image 'vbatts/pandoc:1.16.0.2-1.fc24' locally
1.16.0.2-1.fc24: Pulling from vbatts/pandoc
Status: Downloaded newer image for vbatts/pandoc:1.16.0.2-1.fc24
pandoc: Duplicate link reference `[rfc1952]' "source" (line 1044, column 1)
! Package inputenc Error: Unicode char \u8:₀ not set up for use with LaTeX.
See the inputenc package documentation for explanation.
Type  H <return>  for immediate help.
 ...                                              
                                                  
l.1840 \end{verbatim}
Try running pandoc with --latex-engine=xelatex.
pandoc: Error producing PDF
make: *** [output//oci-image-spec.pdf] Error 43
The command "make docs" exited with 2.
```

assigning this to myself to get pandoc playing nice with the characters introduced in @stevvooe chainID PR

--- a/Makefile
+++ b/Makefile
@@ -13,7 +13,7 @@ ifeq "$(strip $(PANDOC))" ''
 			-v $(shell pwd)/$(OUTPUT_DIRNAME)/:/$(OUTPUT_DIRNAME)/ \
 			-u $(shell id -u) \
 			--workdir /input \
-			docker.io/vbatts/pandoc:1.16.0.2-1.fc24
+			docker.io/vbatts/pandoc:1.17.0.3-2.fc25.x86_64
 		PANDOC_SRC := /input/
 		PANDOC_DST := /
 	endif
@@ -65,7 +65,7 @@ $(OUTPUT_DIRNAME)/$(DOC_FILENAME).pdf: $(DOC_FILES) $(FIGURE_FILES)
 else
 $(OUTPUT_DIRNAME)/$(DOC_FILENAME).pdf: $(DOC_FILES) $(FIGURE_FILES)
 	@mkdir -p $(OUTPUT_DIRNAME)/ && \
-	$(PANDOC) -f markdown_github -t latex -o $(PANDOC_DST)$@ $(patsubst %,$(PANDOC_SRC)%,$(DOC_FILES))
+	$(PANDOC) -f markdown_github -t latex --latex-engine=xelatex -o $(PANDOC_DST)$@ $(patsubst %,$(PANDOC_SRC)%,$(DOC_FILES))
 	ls -sh $(shell readlink -f $@)
 
 $(OUTPUT_DIRNAME)/$(DOC_FILENAME).html: $(DOC_FILES) $(FIGURE_FILES)

